From e328a020645899f25d0afccc196928783d6b59d1 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Thu, 18 Aug 2016 19:08:02 +0200
Subject: [PATCH] kio: fix KFileMetaInfo without strigi

Provide also empty implementations of the two methods preferredGroups()
and supportedGroups() of the KFileMetaInfo class, otherwise they are
left unimplemented when building kdelibs without strigi.
---
 kio/kio/kfilemetainfo.cpp | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/kio/kio/kfilemetainfo.cpp b/kio/kio/kfilemetainfo.cpp
index cdda72f..e66a6a8 100644
--- a/kio/kio/kfilemetainfo.cpp
+++ b/kio/kio/kfilemetainfo.cpp
@@ -479,6 +479,20 @@ QStringList KFileMetaInfo::supportedKeys() const
 {
     return QStringList();
 }
+
+#ifndef KDE_NO_DEPRECATED
+KFileMetaInfoGroupList KFileMetaInfo::preferredGroups() const
+{
+    return KFileMetaInfoGroupList();
+}
+#endif
+
+#ifndef KDE_NO_DEPRECATED
+KFileMetaInfoGroupList KFileMetaInfo::supportedGroups() const
+{
+    return KFileMetaInfoGroupList();
+}
+#endif
 #endif
 
 KFileMetaInfoItemList KFileMetaInfoGroup::items() const
-- 
2.8.1

